projects/core/src/features-config/utils/feature-config-utils.ts

```diff
@@ -21,7 +24,9 @@ function isInLevel(level, version) {
 
 export function isFeatureLevel(config: unknown, level: string): boolean {
   if (isFeatureConfig(config)) {
-    return isInLevel(config.features.level, level);
+    return level.indexOf('!') !== -1
```

Comment:
We should check for ! at the beginning, so either ` return level.indexOf('!') === 0` or even simpler `level[0] === '!'`